pkg/accounting/accounting.go

```diff
@@ -1133,7 +1140,8 @@ func (a *Accounting) NotifyRefreshmentSent(peer swarm.Address, attemptedAmount,
 			a.metrics.AccountingDisconnectsEnforceRefreshCount.Inc()
 			_ = a.blocklist(peer, 1, "failed to refresh")
 		}
-		a.logger.Error(receivedError, "notifyrefreshmentsent failed to refresh")
+		a.logger.Debug("notifyrefreshmentsent failed to refresh", "error", receivedError)
+		a.logger.Error(nil, "notifyrefreshmentsent failed to refresh")
```

Comment:
Ditto again, but I'll quit mentioning it at this point as it all hinges on why the error (receivedError in this case) is not provided to the logger.Error invocation.

Reply:
so umm if you check out other packages you will see that this is kind of a standard where debug contains all of the information but err log contains a general error. I am not sure why we did it the way we did it (could be that we do not want to leak unnecessary info in error, @mrekucci would be better person to explain this) but changing the way its done within pr will no longer look like a standard anymore and maybe if it is required a better way should be another pr focused on refactoring all of those erros. 
Please have a look
https://github.com/ethersphere/bee/blob/master/pkg/api/balances.go#L77
https://github.com/ethersphere/bee/blob/master/pkg/api/stewardship.go#L56
https://github.com/ethersphere/bee/blob/master/pkg/api/postage.go#L101

Reply:
I've seen that tendency in the other packages and quite frankly don't really understand it.   I don't see how the err itself would leak info, but it certainly goes a long way to knowing what's going on without cranking the verbosity up to debug.

But I'm not one to fight city hall, and the PR definitely fixes the original issue, so I'll mark it approved.

Reply:
ill discuss this with @mrekucci once he gets back from vacation. Ty :)

Reply:
I also would like to know why we are doing this :) ? The API has a lot of these.

Reply:
@mrekucci why are we doing things this way?